### PR TITLE
update tinylicious port

### LINKED ISSUE #5415: Users want to config tinylicious and launch it on a port other than 3000
This is the copy of a question asked on stackoverflow:

https://stackoverflow.com/questions/66536190/can-the-tinylicious-server-be-launched-at-a-port-other-than-3000?noredirect=1#comment117623175_66536190

Can the tinylicious server be launched at a port other than 3000? I've tried something like "PORT=4100 tinylicious" and I can see the terminal log saying:

`@federation/shell-app: [1] info: Listening on port 4100 {"label":"winston","timestamp":"2021-03-08T19:23:37.861Z"}
`

but later it fails within my code, indicating something went wrong with the service call:

```
main.js:15815 ERROR TypeError: Cannot read property 'shapeClicked' of undefined
    at Layer.onClick [as zzClickFunc] (collabmap.component.js:45)
    at JS:24817
    at Array.<anonymous> (JS:8190)
    at window.<computed> (JS:1111)
    at Object.<anonymous> (JS:51778)
    at j (JS:51777)
```
and indeed, the Network tab reveals it's still posting via 3000:

```
Request URL: http://localhost:3000/documents/tinylicious
Referrer Policy: strict-origin-when-cross-origin
```

I know tinylicious is not the full Fluid server and it's just for testing purposes, so it might have been hardwired to 3000, but maybe someone has an idea how to launch it on a different port.

### PR DESCRIPTION
**Pull request by micahgodbolt**
helps resolve #5415

does not make configurable, but moves away from the common :3000 port

### CODE REVIEW

File: packages/drivers/tinylicious-driver/src/insecureTinyliciousUrlResolver.ts [hunk 1]
```diff
@@ -12,6 +12,9 @@ import {
 import { ITokenClaims } from "@fluidframework/protocol-definitions";
 import { KJUR as jsrsasign } from "jsrsasign";
 import { v4 as uuid } from "uuid";
+
+const defaultPort = "35843";
```
Comment (ChumpChief) [inline packages/drivers/tinylicious-driver/src/insecureTinyliciousUrlResolver.ts:16]: Export this as `tinyliciousDefaultPort` and use in the other `packages/*` packages probably

**Review by tylerbutler — APPROVED** [review]
🎸

**Review by tanviraumi — APPROVED** [review]

File: packages/drivers/tinylicious-driver/src/insecureTinyliciousUrlResolver.ts [hunk 2]
```diff
@@ -20,19 +23,19 @@ import { v4 as uuid } from "uuid";
  */
 export class InsecureTinyliciousUrlResolver implements IUrlResolver {
```
Comment (anthony-murphy) [inline packages/drivers/tinylicious-driver/src/insecureTinyliciousUrlResolver.ts:24]: why not just add a construcutor and a param to make the whole address configurable? you can default to localhost and that port

File: server/tinylicious/src/resourcesFactory.ts [hunk 3]
```diff
@@ -26,7 +26,7 @@ import {
 export class TinyliciousResourcesFactory implements utils.IResourcesFactory<TinyliciousResources> {
     public async create(config: Provider): Promise<TinyliciousResources> {
         // Pull in the default port off the config
-        const port = utils.normalizePort(process.env.PORT || "3000");
+        const port = utils.normalizePort(process.env.PORT || "35843");
```
Comment (anthony-murphy) [inline server/tinylicious/src/resourcesFactory.ts:29]: you need to stage this. first make this change, release, and consume in client with the driver change

Comment (micahgodbolt) [reply server/tinylicious/src/resourcesFactory.ts:29]: ok, PR changed to only change tinylicious

**Review by anthony-murphy — APPROVED** [review]

**Pull request merged**